code/code.go

```diff
@@ -0,0 +1,30 @@
+package code
+
+import api "github.com/tigrisdata/tigris-client-go/api/server/v1"
+
+const (
+	OK                 = api.Code_OK
```

Comment:
Can we have http status code as a field as well in TigrisError?

Reply:
It's now available as a method of TigrisError
and can be used like `c := te.GetHTTPCode()`